packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -355,6 +373,7 @@ export function defineNuxtLink (options: NuxtLinkOptions) {
             replace: props.replace,
             ariaCurrentValue: props.ariaCurrentValue,
             custom: props.custom,
+            onMouseenter: props.prefetchMode === 'hover' && checkShouldPrefetch() ? prefetch.bind(null, undefined) : undefined,
```

Comment:
Would it be better to use `onPointerEnter`? As far as I know, `MouseEvent` is not supported on mobile devices, while `PointerEvent`, which inherits from `MouseEvent`, supports input methods such as mouse, stylus, and touch.

Reply:
@Mini-ghost Sounds interesting, I did not take mobile into account because the common usage of web apps on mobile won't benefit from this since users will click immediately while on a desktop is required to hover before clicking so prefetching that way is handy. If we can support prefetching on hover while using a mouse on mobile devices I'd be happy to implement it in the proposed way. Thanks - I appreciate it!

Reply:
Also, it makes me think a bit about users navigating across a page using a keyboard; maybe prefetching on focus would be a good option. What do you think @Mini-ghost @danielroe

Reply:
Agreed. What about 'interaction' as the option and whatever is most generic events we can listen to.

Reply:
You are correct. I tested it, and unless intentionally triggered, `hover` is indeed difficult to activate on mobile devices. However, I still recommend using `onPointerenter` as it does not have any negative impact.

Additionally, having `interaction` as one of the options is fantastic! This seems to encompass both `onPointerenter` and `onFocus` (or `onFocusin`).

Reply:
@danielroe @Mini-ghost The `interaction` options sounds fine, but then it made me think if having two options `hover` and `interaction` can be a bit confused for customers of the API, and would require a vivid description in docs describing what exactly does interaction mean since it's a broad term. Ultimately I think that API like 
```tsx
  prefetchTriggers?: Partial<{
    visibility: boolean
    focus: boolean
    hover: boolean
  }>
```
With defaults:
```tsx
  prefetchTriggers: {
    visibility: true
    focus: false
    hover: false
  }
```
might be a good option. What do you think about this shift?